cmd/headscale/cli/utils.go

```diff
@@ -108,10 +108,23 @@ func GetDNSConfig() *tailcfg.DNSConfig {
 			dnsConfig.Domains = viper.GetStringSlice("dns_config.domains")
 		}
 
-		return dnsConfig
+		if len(dnsConfig.Nameservers) > 0 {
+			if viper.IsSet("dns_config.magic_dns") {
+				dnsConfig.Proxied = viper.GetBool("dns_config.magic_dns")
+			}
+		}
+
+		var baseDomain string
+		if viper.IsSet("dns_config.base_domain") {
+			baseDomain = viper.GetString("dns_config.base_domain")
+		} else {
+			baseDomain = "headscale.net" // does not really matter when MagicDNS is not enabled
```

Comment:
Since we dont own (I think?) this domain, can we swap it out for headscale.font.eu or something? I feel like we should control the placeholder.

Reply:
I do own that domain !! haha